src/cursor-doc/token-cursor.ts

```diff
@@ -670,6 +670,15 @@ export class LispTokenCursor extends TokenCursor {
         return false;
     }
 
+    /**
+     * Indicates if the current token is in a comment line
+     */
+    withinComment() {
+        const cursor = this.clone();
+        const token_type = cursor.getToken().type;
+        return token_type === 'comment' || (token_type === 'eol' && cursor.getPrevToken().type === 'comment');
+    }
+
```

Comment:
Now this seems to be in two places. 😄 That was good this time, because I think I know where I prefer it to be. Which is that we, for now, leave it as a method on the cursor itself. We can refactor this later to have a separate module where `withinString`, this one, and some others can live.

Reply:
Oops!